src/tls/settings.rs

```diff
@@ -169,7 +169,11 @@ impl TlsSettings {
             context
                 .set_verify_cert_store(store.build())
                 .context(SetVerifyCert)?;
+        } else {
+            #[cfg(windows)]
+            load_windows_certs(context).unwrap();
```

Comment:
@bruceg I am not totally sure if this is the right place but to me this is the spot where we know we want tls to be enabled but there are no ca certs set on the settings. This is at the point where we should attempt to load system certs. Let me know if this thinking is correct.

If so we can then just add a line here for macos and we should be good to go.

Reply:
I think that's right. The `context.set_verify_cert_store` sets the certificate store (as opposed to adding to it), so you would only need to pull in the system store if there is no configured authority.